### PR TITLE
Update zh-cn.yml and zh-TW.yml.

### PR DESCRIPTION
**Pull request by 2016WUJI01**

### CODE REVIEW

**Comment by hanwu85** [comment]
> Also, what is the context of the change in zh-TW?

It is a single Chinese character fix.  But actually in TW chinese, it is the correct one.  Therefore, no need to fix it.

**Comment by 2016WUJI01** [comment]
Sorry for change zh-TW.
This pr changed this page: https://www.worldcubeassociation.org/teams-committees-councils

File: config/locales/zh-TW.yml [hunk 1]
```diff
@@ -1224,7 +1224,7 @@ zh-TW:
       title: WCA團隊、委員會和理事會
       #original_hash: d54d953
       description: >-
-        WCA的團隊和委員會是整個組織的一部分並負責日常的事務才能讓整個組織運作。這些團隊是由WCA成員包含WCA代表和參賽者所組成的。每一個團隊會都有一個領導者，他們的職務是確保整個團隊的正常運作以及帶領其他團隊成員。理事會是一個特殊的團隊，他們的工作是擔任WCA董事會諮詢的腳色，由社群成員組成而非WCA員工。當有委員會、團隊、理事會的空缺時，如何申請的資訊將會公布在首頁。
+        WCA的團隊和委員會是整個組織的一部分並負責日常的事務才能讓整個組織運作。這些團隊是由WCA成員包含WCA代表和參賽者所組成的。每一個團隊會都有一個領導者，他們的職務是確保整個團隊的正常運作以及帶領其他團隊成員。理事會是一個特殊的團隊，他們的工作是擔任WCA董事會諮詢的角色，由社群成員組成而非WCA員工。當有委員會、團隊、理事會的空缺時，如何申請的資訊將會公布在首頁。
```
Comment (gregorbg) [inline config/locales/zh-TW.yml:1227]: Please undo this change as per the previous discussion.

**Review by gregorbg — CHANGES_REQUESTED** [review]

**Comment by 2016WUJI01** [comment]
> In that case, please undo the change in `zh-TW.yml`.
> 
> Also, I would really prefer if you could get _any_ Chinese Delegate to approve you as a translator. That way, we don't have to discuss and get reviews under every single individual PR. Is there any chance that you can get any Chinese Delegate to confirm you?

I can't find any Chinese Delegate to vouch for me as a translator. Might need @notrichardpeng for future PRs.

**Comment by notrichardpeng** [comment]
> > In that case, please undo the change in `zh-TW.yml`.
> > Also, I would really prefer if you could get _any_ Chinese Delegate to approve you as a translator. That way, we don't have to discuss and get reviews under every single individual PR. Is there any chance that you can get any Chinese Delegate to confirm you?
> 
> I can't find any Chinese Delegate to vouch for me as a translator. Might need @notrichardpeng for future PRs.

I don't mind reviewing translations, but have you reached out to any of them?

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -1182,6 +1182,90 @@ zh-CN:
...
+        #original_hash: 8f32116
+        wqac: WCA质量保证委员会
+        #original_hash: 3da1e85
+        wrc: WCA规则委员会
+        #original_hash: 67be768
+        wrt: WCA成绩组
+        #original_hash: a14bc62
+        wst: WCA软件组
+        #original_hash: 58765c7
+        wsot: WCA竞技组织团队
```
Comment (notrichardpeng) [inline config/locales/zh-CN.yml:1229]: 既然是负责奥林匹克，"体育组织" 会不会好一点

File: config/locales/zh-CN.yml [hunk 3]
```diff
@@ -1182,6 +1182,90 @@ zh-CN:
...
+      officers_description: >-
+        WCA董事负责负责处理与WCA非营利地位相关的任务。这些董事由WCA董事会选举产生。执行董事是WCA的首席执行官，主席主持所有董事会会议，秘书负责维护组织的文件，财务主管管理WCA的财务事务。
+      #original_hash: e0d37a2
+      board_description: WCA董事会负责领导整个组织，并履行其他团队、委员会和理事会未履行的任何职责。
+    teams_committees_councils:
+      #original_hash: 0d62207
+      title: WCA团队、委员会和理事会
+      #original_hash: d54d953
+      description: >-
+        WCA的团队和委员会是组织不可分割的一部分，执行保持其运作所需的日常任务。这些委员会由WCA社区成员组成，包括WCA代表和竞争对手。每个团队都有一个领导者，其作用是确保团队的正常运作并指导其他团队成员。理事会是一种特殊类型的团队，为WCA董事会发挥咨询作用，由社区成员组成，而不是WCA工作人员。当有委员会、团队或理事会空缺时，有关如何申请的信息会发布在首页上。
```
Comment (notrichardpeng) [inline config/locales/zh-CN.yml:1202]: "竞争对手" 换成 “参赛人员” 应该更好一些

**Comment by 2016WUJI01** [comment]
> reached



> > > In that case, please undo the change in `zh-TW.yml`.
> > > Also, I would really prefer if you could get _any_ Chinese Delegate to approve you as a translator. That way, we don't have to discuss and get reviews under every single individual PR. Is there any chance that you can get any Chinese Delegate to confirm you?
> > 
> > 
> > I can't find any Chinese Delegate to vouch for me as a translator. Might need @notrichardpeng for future PRs.
> 
> I don't mind reviewing translations, but have you reached out to any of them?

Yep, I tried to contact one of them @Baiqiang , but there was no reply.

File: config/locales/zh-CN.yml [hunk 4]
```diff
@@ -1182,6 +1182,90 @@ zh-CN:
...
+          该委员会负责支持和促进WCA内持续的质量改进，以及在全球范围内应用质量标准，以确保一致的高质量流程，WCA工作人员，区域组织，比赛组织者和工作人员。
+        #original_hash: a1260bb
+        wrc: >-
+          该组负责与WCA规则的应用、改进和发展有关的所有问题。他们支持WCA代表处理比赛中发生的任何类型的程序性问题，并决定未解决和未发现的事件。
+        #original_hash: 7bc588b
+        wrt: 该组负责管理WCA数据库中的所有数据，最重要的是比赛成绩和选手个人数据。
+        #original_hash: ec961e4
+        wst: 该组负责管理WCA软件（官网、打乱程序、管理工具等）。
+        #original_hash: 8f888d6
+        wsot: 该团队负责监督和协助WCA成为一个国际认可的竞技组织。
```
Comment (notrichardpeng) [inline config/locales/zh-CN.yml:1266]: 同上，“体育组织”？

**Review by notrichardpeng — CHANGES_REQUESTED** [review]

**Comment by notrichardpeng** [comment]
> > reached
> 
> > > > In that case, please undo the change in `zh-TW.yml`.
> > > > Also, I would really prefer if you could get _any_ Chinese Delegate to approve you as a translator. That way, we don't have to discuss and get reviews under every single individual PR. Is there any chance that you can get any Chinese Delegate to confirm you?
> > > 
> > > 
> > > I can't find any Chinese Delegate to vouch for me as a translator. Might need @notrichardpeng for future PRs.
> > 
> > 
> > I don't mind reviewing translations, but have you reached out to any of them?
> 
> Yep, I tried to contact one of them @Baiqiang , but there was no reply.

Perhaps look on WCA and see which delegates from China are actively organizing, they might have a better chance of responding.

**Review by notrichardpeng — APPROVED** [review]

**Pull request merged**